sycophancy_analysis/evaluate_llm_judges/evaluate_single_model.py

```
@@ -0,0 +1,504 @@
+    total_rows = len(gt_df)
+    
+    # Setup progress tracking
+    start_time = time.time()
+    progress_bar = tqdm(total=total_rows, desc="Evaluating responses", unit="response")
+    
+    # Track processing statistics
+    processed_count = 0
+    error_count = 0
+    
+    for i, r in enumerate(gt_df.itertuples(index=False)):
+        pid = getattr(r, "prompt_id")
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)